get/project_config.go

```diff
@@ -0,0 +1,116 @@
+package get
+
+import (
+	"context"
+	"io"
+	"strconv"
+	"text/tabwriter"
+	"time"
+
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+	"github.com/ninech/nctl/internal/format"
+	"k8s.io/apimachinery/pkg/util/duration"
+)
+
+type configsCmd struct {
+	out io.Writer
+}
+
+func (cmd *configsCmd) Run(ctx context.Context, client *api.Client, get *Cmd) error {
+	projectConfigList := &apps.ProjectConfigList{}
+
+	var opts []listOpt
+
+	if !get.AllNamespaces {
+		opts = []listOpt{matchName(client.Namespace)}
+	}
+
+	if err := get.list(ctx, client, projectConfigList, opts...); err != nil {
+		return err
+	}
+
+	filterProjectConfigList(projectConfigList)
+
+	if len(projectConfigList.Items) == 0 {
+		printEmptyMessage(apps.ProjectConfigKind, client.Namespace)
+		return nil
+	}
+
+	switch get.Output {
+	case full:
+		return printProjectConfigs(projectConfigList.Items, get, defaultOut(cmd.out), true)
+	case noHeader:
+		return printProjectConfigs(projectConfigList.Items, get, defaultOut(cmd.out), false)
+	case yamlOut:
+		return format.PrettyPrintObjects(projectConfigList.GetItems(), format.PrintOpts{Out: defaultOut(cmd.out)})
+	}
+
+	return nil
+}
+
+// filterProjectConfigList filters out everything except the ProjectConfig
+// resource whose Name is equal to its Namespace (as defined in the project
+// documentation, a valid Project Configuration has Name=Namespace).
+// If no such resource is found, func returns an empty slice.
+func filterProjectConfigList(configList *apps.ProjectConfigList) {
```

Comment:
Do we really need this? We will have a webhook in place not allowing to create any project configs like that anyways.